platform-sdk/docs/core/dynamic-address-book/roster/glossary-of-terms.md

```diff
@@ -0,0 +1,100 @@
+# Roster Glossary of Terms
+![Event Window Terminology](./EventWindowTerminology.drawio.svg)
+![Roster Terminology Diagram](./RosterTerminologyDiagram.drawio.svg)
+## Consensus Terminology
+
+### Latest Consensus Round
+
+The most recent round to have reached consensus.  (equal to the pending consensus round - 1)
+
+### Pending Consensus Round
+
+The round that the consensus algorithm is currently attempting to reach consensus on. (equal to the latest 
+consensus round + 1)
+
+### Rounds Non-Ancient
+
+A setting. The number of rounds that have reached consensus that are not ancient.  Currently set to 26 in all 
+environments. 
+
+### Ancient
+
+An event is ancient if its `birthRound` is less than or equal to the`latest consensus round` - `rounds non-ancient`.
+
+### Minimum Round Non-Ancient
+
+`minRoundNonAncient` is the minimum round that is considered non-ancient.  This is equal to the `pending consensus 
+round` - `rounds non-ancient`.
+
+### Non-Ancient Event Window
+
+Specifies the window of events that are being considered in consensus.  The window specified the following:
+* `latestConsensusRound` - the latest round that has reached consensus.
+* `minRoundNonAncient` - the minimum round that is considered non-ancient.
+* `minGenNonAncient` - the minimum generation that is considered non-ancient.  (This is deprecated). 
```

Comment:
Well, not deprecated yet. It will become deprecated, and at that time we can update the documentation to remove any mention of generation being used to determine ancient status.